database_schemas/cassandra/benchflow.cql

```diff
@@ -23,14 +23,26 @@ CREATE TABLE database_sizes (
   trial_id text,
   dbms text,
   database_name text,
-  database_table text,
   size bigint,
   PRIMARY KEY ((trial_id, experiment_id), dbms)
 );
 
+CREATE TABLE io_data (
+  experiment_id text,
+  trial_id text,
+  container_properties_id uuid,
+  device text,
+  writes bigint,
+  reads bigint,
+  sync bigint,
+  async bigint,
+  total bigint,
+  PRIMARY KEY ((trial_id, experiment_id), container_properties_id)
+);
+
 CREATE TABLE environment_data (
   environment_data_id uuid,
-  container_properties_id uuid,
+  container_id text,
```

Comment:
Yes, either we have a dedicated collector to collect the discussed information, or we rely on one of the collectors that interact with the Docker socket already, and currently it is only the stats collector. Moreover I also think that the fact that the stats collector, also collects the metadata for which it collects the statistics, it is consistent. We might change our mind at some point, but currently I think it is a good idea in the system we are building.